customer_data_platform/public_api/customer_data_platform_profiles_api.yaml

```diff
@@ -0,0 +1,49 @@
+openapi: 3.1.0
+info:
+  title: CDP Profiles API Documentation
+  version: 0.0.0
+  description: This is the documentation of the CDP Profiles API service.
+servers:
+  - url: https://organization.piwik.pro
+paths:
+  /api/cdp/profiles/public/v1/app/{app_id}/profile/{cookie_id}/audiences:
+    get:
+      summary: Fetch audiences the profile belongs to
+      tags:
+        - profiles
+      security: []
+      operationId: get-profile-audiences
+      description: Get audiences the profile belongs to
+      parameters:
+        - $ref: "components.yaml#/components/parameters/AppId"
+        - in: path
+          name: cookie_id
+          description: Cookie ID stored in cookies
+          schema:
+            type: string
+            example: "3a0133748b3e524c"
+          required: true
+      responses:
+        200:
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: "#/components/schemas/ProfileAudienceIdList"
+          description: An array of last activation log entries
```

Comment:
This seems wrong

Reply:
Changed. Also changed in original docs: https://github.com/PiwikPRO/PPMS-CDP/pull/341